doc/plugin_server_notifier_k8sbundle.md

```diff
@@ -134,3 +135,49 @@ server to
         }
     }
 ```
+
+### Multipe clusters
```

Comment:
I think the second one looks better. Is the `namespace`, `config_map`, etc. fields expected to be different across clusters? If not, it can condense down to a list of kubeconfig files.

Reply:
That's a great idea. Assuming we agree on not making the kube config file path required, what do you think of this instead? 

---


Same cluster multiple namespaces:

```
plugin_data {
  namespace = "spire"
  clusters = [
  {
    namespace = "istio-system"
  },
  {
    namespace = "default"
  }
  ]
}
```
Which is the same as:
```
plugin_data {  
  clusters = [
  {
    # spire namespace due to default values
  },
  {
    namespace = "istio-system"
  },
  {
    namespace = "default"
  }
  ]
}
```

Or multiple clusters:

```
plugin_data {  
  clusters = [
  {
    # this is local cluster with default values
  },
  {
    kube_config_file_path = "/cluster2/file/path"
  },
  {
    kube_config_file_path = "/cluster3/file/path"
  }
  ]
}
```